### PR TITLE
Initial commit

### PR DESCRIPTION
**Pull request by dlouvier**
Initial commit.

- Prepare workflows
- Add [Readme](https://github.com/getmoss/datadog-costs-exporter/blob/initial-commit/README.md)
- Initial working code (not 100% covered by tests)

### CODE REVIEW

File: go/main.go [hunk 1]
```diff
@@ -0,0 +1,230 @@
...
+					{Name: "charge_type", Type: bigquery.StringFieldType},
+					{Name: "cost", Type: bigquery.FloatFieldType},
+					{Name: "product_name", Type: bigquery.StringFieldType},
+				}},
+		},
+	}
+
+	for _, t := range []string{args.TABLE_NAME_ID, args.TABLE_NAME_ID + args.TEMP_TABLE_SUFFIX} {
+		if _, err := b.Dataset(args.DATASET_ID).Table(t).Metadata(ctx); err != nil {
+			if err := b.Dataset(args.DATASET_ID).Table(t).Create(ctx, tableMetaData); err != nil {
```
Comment (raj-saxena) [inline go/main.go:101]: Seems like you are always creating the tables. 
Shouldn't this first check if the table already exists and skip if it does?

File: go/main.go [hunk 2]
```diff
@@ -0,0 +1,230 @@
...
+	Charges   []ChargebackBreakdown `bigquery:"charges"`
+}
+
+type ChargebackBreakdown struct {
+	ChargeType  string  `bigquery:"charge_type"`
+	Cost        float64 `bigquery:"cost"`
+	ProductName string  `bigquery:"product_name"`
+}
+
+func (c *CostByOrg) Save() (map[string]bigquery.Value, string, error) {
```
[the remark this answers is not in the document]
Comment (dlouvier) [reply go/main.go:44]: This is required by bigquery.table.put()

File: go/main.go (hunk 1 above)
Comment (dlouvier) [reply go/main.go:101]: Not sure if I understood you, It only checks if table exist when the program starts once. If it doesn't, it creates it.

The way to verify if a table exist, it is to check .Metadata(), if it returns an error, the table doesn't exist. There is no such method like .isTable() or .exists()

Comment (raj-saxena) [reply go/main.go:101]: > The way to verify if a table exist, it is to check .Metadata(), ...

Ah okay. All good then, ignore.

**Review by raj-saxena — APPROVED** [review]

**Pull request merged**